cookbook/form/create_form_type_extension.rst

```diff
@@ -319,3 +311,15 @@ next to the file field. For example::
 
 When displaying the form, if the underlying model has already been associated
 with an image, you will see it displayed next to the file input.
+
+Generic Form Type Extensions
+----------------------------
+
+Although it is not possible to have a form type extension applying to all form
+types, **most** form types natively available in Symfony
+(:doc:`/reference/forms/types`) inherit from the ``form`` form type. Thus, a
+form type extension applying to ``form`` would apply to all of these.
+
+A notable exception are the ``button`` from types. Plus, keep in mind that a custom
+form type which doesn't inherit neither ``form`` nor ``button`` could always be
+created.
```

Comment:
Indeed, having a positively worded documention would be much better.

I also tried to reference back the example provided in the bullet point list to provide consistence.